Core/PixelExperiment.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  PixelExperiment.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public enum PixelExperiment: String, CaseIterable {
+
+    fileprivate static let logic = PixelExperimentLogic {
+        Pixel.fire(pixel: $0)
+    }
+
+    /// When `cohort` is accessed for the first time after the experiment is installed with `install()`,
+    ///  allocate and return a cohort.  Subsequently, return the same cohort.
+    public static var cohort: PixelExperiment? {
+        logic.cohort
+    }
+
+    static var isExperimentInstalled: Bool {
+        return logic.isInstalled
+    }
+
+    static var allocatedCohortDoesNotMatchCurrentCohorts: Bool {
+        guard let allocatedCohort = logic.allocatedCohort else { return false }
+        if PixelExperiment(rawValue: allocatedCohort) == nil {
+            return true
+        }
+        return false
+    }
+
+    /// Enables this experiment for new users when called from the new installation path.
+    public static func install() {
+        // Disable the experiment until all other experiments are finished
+        // logic.install()
+    }
+
+    static func cleanup() {
+        logic.cleanup()
+    }
+
+    // These are the variants. Rename or add/remove them as needed.  If you change the string value
+    //  remember to keep it clear for privacy triage.
+    case control
+    case newSettings = "variant1"
```

Comment:
This will result in:

/t/pixel?cohort=variant1

or 

/t/pixel?cohort=control

depending on which cohort they end up in.  I'd recommend dropping the "variant1" so that it just ends up as "newSettings".  Mainly from a privacy triage perspective.  If you've already agreed that 'variant1' is good enough that's fine.

Reply:
Sounds good 👍